packages/core/test/Serialization/PlutusData.test.ts

```diff
@@ -167,6 +167,14 @@ describe('PlutusData', () => {
 
       expect(data.toCbor()).toEqual('a10102');
     });
+
+    it('can find an element in a plutus map with key', () => {
+      const data = new Serialization.PlutusMap();
+
+      data.insert(Serialization.PlutusData.newInteger(1n), Serialization.PlutusData.newInteger(2n));
+
+      expect(data.get(Serialization.PlutusData.newInteger(1n))).toEqual(Serialization.PlutusData.newInteger(2n));
```

Comment:
Would be great to add this assertion to make sure that 1n and 2n PlutusData are not deepEquals. Wouldn't be a bad idea to add `equals` method to all PlutusData classes to be sure it works as expected with all kinds.
```ts
expect(data.get(Serialization.PlutusData.newInteger(2n))).toBeUndefined();
```